galaxy_ng/app/api/ui/serializers/user.py

```diff
@@ -0,0 +1,80 @@
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from galaxy_ng.app.api import permissions
+
+from galaxy_ng.app.models import auth as auth_models
+
+
+class GroupSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = auth_models.Group
+        fields = (
+            'id',
+            'name'
+        )
+
+
+class UserSerializer(serializers.ModelSerializer):
+    password_confirm = serializers.CharField(write_only=True, allow_blank=True, required=False)
+
+    class Meta:
+        model = auth_models.User
+        fields = (
+            'id',
+            'username',
+            'first_name',
+            'last_name',
+            'email',
+            'groups',
+            'password',
+            'password_confirm'
+        )
+        extra_kwargs = {
+            'password': {'write_only': True, 'allow_blank': True, 'required': False}
```

Comment:
You could've add `min_length=` here instead of writing custom validation.

Reply:
UPD: Password validation is configured globally for Django projects via `AUTH_PASSWORD_VALIDATORS` settings. In addition to that Django provides builtin facilities to validate user password [1]
This is the preferred way which you should use to validate user passwords.


[1] https://docs.djangoproject.com/en/3.0/topics/auth/passwords/#module-django.contrib.auth.password_validation

Reply:
Adding this might break the initial data since the default admin password will be invalid. I'll look into it

Reply:
also, I believe anything we put in settings can potentially get overridden by other pulp plugins

Reply:
My point is that we should rely on a password policy configured at the project level, because we use `django.auth`.